pkg/kobject/kobject.go

```diff
@@ -107,3 +108,15 @@ type Ports struct {
 	HostIP        string
 	Protocol      api.Protocol
 }
+
+// Volumes holds the volume struct of container
+type Volumes struct {
+	SvcName    string
+	MountPath  string
+	VFrom      string
+	VolumeName string
+	Host       string
+	Container  string
+	Mode       string
+	PVCName    string
+}
```

Comment:
This is a bit confusing for me also. Maybe it would be nice to have some comments describing what some of the variables are for.